src/main/java/com/github/dockerjava/api/model/Network.java

```diff
@@ -65,6 +71,14 @@ public Ipam getIpam() {
         return options;
     }
 
+    public boolean isAttachable() {
+        return attachable == null ? false : attachable;
```

Comment:
please return raw value, on end-user code you can wrap it simply with `BooleanUtils`
null in return value means that there is no value. No value != false from req/rsp view point.